tools/python/paladin_claiming/recycle_bribes.py

```diff
@@ -0,0 +1,202 @@
+import json
+from dotenv import load_dotenv
+from typing import List
+import sys
+from pathlib import Path
+import os
+from datetime import datetime
+from decimal import Decimal
+
+sys.path.append(str(Path(__file__).parent.parent))
+
+import requests
+from web3 import Web3
+from helpers.hh_bribs import get_hh_aura_target
+from bal_addresses import AddrBook
+from bal_tools.safe_tx_builder import SafeTxBuilder, SafeContract
+from bal_tools import Web3RpcByChain
+
+
+load_dotenv()
+
+SCRIPT_DIR = Path(__file__).parent
+CLAIM_OUTPUT_DIR = SCRIPT_DIR / "claim_output"
+PAYLOAD_DIR = SCRIPT_DIR / "payload"
+
+CLAIM_OUTPUT_DIR.mkdir(parents=True, exist_ok=True)
+PAYLOAD_DIR.mkdir(parents=True, exist_ok=True)
+
+CURRENT_DATE = datetime.now().strftime("%Y-%m-%d")
+
+w3_by_chain = Web3RpcByChain(os.getenv("DRPC_KEY"))
+flatbook_mainnet = AddrBook("mainnet").flatbook
+flatbook_arb = AddrBook("arbitrum").flatbook
+
+omni_safe = flatbook_mainnet["multisigs/vote_incentive_recycling"]
+
+CHAIN_ADDRS = {
+    "mainnet": {
+        "paladin_distributor": flatbook_mainnet["paladin/DistributorV2"],
+        "bribe_vault": flatbook_mainnet["hidden_hand2/bribe_vault"],
+        "bal_briber": flatbook_mainnet["hidden_hand2/balancer_briber"],
+    },
+    # 0x089154A7E4C562d5998AB3D7Ca57B504A8912482
+    "arbitrum": {
+        "paladin_distributor": flatbook_arb["paladin/DistributorV1"],
+        "bribe_vault": flatbook_arb["hidden_hand2/bribe_vault"],
+        "bal_briber": flatbook_arb["hidden_hand2/bal_briber"],
+    },
+}
+
+
+paladin_distributor_abi = json.load(
+    open("tools/python/abis/MultiMerkleDistributorV2.json")
```

Comment:
this is confusing. the requirements.txt is `multisig-ops/tools/python/`, so i would consider that to be the root dir and place where the env lives. however, opening it like this implicates we are running from `multisig-ops/` instead?

Reply:
i refactored the path handling to work the same way as the aura_snapshot_voting/vote.py script - should make it more clear and will ensure it works when run via gh action - 97108d8

Reply:
works great now!